workflow/sample_files/base-bldgtype-multifamily-shared-ground-loop-ground-to-air-heat-pump.xml

```diff
@@ -254,11 +254,16 @@
                 <Units>COP</Units>
                 <Value>3.6</Value>
               </AnnualHeatingEfficiency>
+              <AttachedToGeothermalLoop idref='GeothermalLoop1'/>
               <extension>
                 <PumpPowerWattsPerTon>0.0</PumpPowerWattsPerTon>
                 <SharedLoopWatts>600.0</SharedLoopWatts>
               </extension>
             </HeatPump>
+            <GeothermalLoop>
+              <SystemIdentifier id='GeothermalLoop1'/>
+              <LoopConfiguration>vertical</LoopConfiguration>
+            </GeothermalLoop>
```

Comment:
Why are all of these files changing? They shouldn't be, considering the geothermal loop is optional.

Reply:
There isn't a "none" option, so the new `set_geothermal_loop` method can pass a whole set of `nil` values into `hpxml.geothermal_loops.add(...)`.

Reply:
Perhaps I can add `geothermal_loop_borefield_configuration_choices << 'none'`, and make `geothermal_loop_borefield_configuration` a required argument that defaults to `'none'`.

Reply:
I suggest just not calling `hpxml.geothermal_loops.add(...)` if all the values are nil.

Reply:
Should `hpxml_defaults.rb` add a (defaulted) geothermal loop if it doesn't exist?

Reply:
Yes, I think it should so the user can inspect the defaults.

Reply:
I'm going to add a new required `geothermal_loop_configuration` argument that defaults to "none". Its only other choice is "vertical". That way, users can select "vertical" and assign no other geothermal loop arguments, and everything about the geothermal loop can still be defaulted.